generated/__init__.py

```diff
@@ -0,0 +1,13 @@
+from nidaqmx.errors import DaqError, DaqReadError, DaqWriteError, DaqWarning, DaqResourceWarning
```

Comment:
This copies the non-generated files to the `generated/` directory as well. I assume that they will live in `src/` and the codegen or build process will automatically copy them over that that the `generated/` directory contains a fully-formed `nidaqmx` module.

Reply:
Yup, `generated/nidaqmx` should be everything. Some part of the build/release process should copy from `src/nidaqmx` to `generated/nidaqmx` in addition to regenerating all the codegen.

Reply:
Yes, some part of the files which are hand written will be present in `src` and the generator will make sure of copying these files to the `generated` folder as part of the generation